access_test.go

```diff
@@ -74,111 +70,8 @@ func authenticateAccess() string {
 	return cred
 }
 
-func TestSetupInvitedUser(t *testing.T) {
-	initAccessTest(t)
-	tempDirPath, createTempDirCallback := coreTests.CreateTempDirWithCallbackAndAssert(t)
-	defer createTempDirCallback()
-	setEnvCallBack := clientTestUtils.SetEnvWithCallbackAndAssert(t, coreutils.HomeDir, tempDirPath)
-	defer setEnvCallBack()
-	setupServerDetails := &config.ServerDetails{Url: *tests.JfrogUrl, AccessToken: *tests.JfrogAccessToken}
-	encodedCred := encodeConnectionDetails(setupServerDetails, t)
-	setupCmd := coreEnvSetup.NewEnvSetupCommand().SetEncodedConnectionDetails(encodedCred)
-	suffix := setupCmd.SetupAndConfigServer()
-	assert.Empty(t, suffix)
-	configs, err := config.GetAllServersConfigs()
-	assert.NoError(t, err)
-	assert.Equal(t, 1, len(configs))
-	// Verify config values
-	assert.Equal(t, configs[0].Url, *tests.JfrogUrl)
-	assert.Equal(t, *tests.JfrogUrl+"artifactory/", configs[0].ArtifactoryUrl)
-	// Verify token was refreshed
-	assert.NotEqual(t, *tests.JfrogAccessToken, configs[0].AccessToken)
-	assert.NotEmpty(t, configs[0].RefreshToken)
-}
-
-func encodeConnectionDetails(serverDetails *config.ServerDetails, t *testing.T) string {
-	jsonConnectionDetails, err := json.Marshal(serverDetails)
-	assert.NoError(t, err)
-	encoded := base64.StdEncoding.EncodeToString(jsonConnectionDetails)
-	return encoded
-}
-
-func TestRefreshableAccessTokens(t *testing.T) {
```

Comment:
I suggest keeping this test, which covers the [refresh tokens functionality](https://github.com/jfrog/jfrog-cli-core/blob/master/utils/config/tokenrefresh.go).